eip-template.md

```diff
@@ -1,6 +1,7 @@
 ---
 eip: <to be assigned>
 title: <EIP title>
+description: <Simple summary of the EIP>
```

Comment:
Recommend putting this text in here:
> Provide a simplified and layman-accessible explanation of the EIP. Imagine an email subject line, GitHub PR title, or forum post title.

People really struggle with "simple" I have found, and giving them extra guidance is valuable I think.

Reply:
Yes, it should be much better phrased. However "imagine a subject line" is confusing, as I suppose the `title` should be that.

Reply:
Title should be shorter than a subject line.  Title is more like movie title, one or two words.  Subject line is more like a description, about a sentence.

Reply:
Done.